designer/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/RemoteEnvironment.scala

```diff
@@ -205,15 +222,48 @@ trait StandardRemoteEnvironment extends FailFastCirceSupport with RemoteEnvironm
         processName,
         isFragment
       )
-    invokeForSuccess(
-      HttpMethods.POST,
-      List("migrate"),
-      Query.Empty,
-      HttpEntity(migrateScenarioRequest.asJson.noSpaces),
-      List.empty
-    )
+
+    val f: EitherT[Future, NuDesignerError, Unit] = for {
+      remoteNuVersion <- EitherT(fetchRemoteNuVersion)
```

Comment:
why do we have to do it? We are in the Source Nussknacker. We want to migrate the scenario to the Target Nussknacker. I thought we would be sending two things `(source_nu_version, scenario_data_in_source_nu_version_format)`.  At the target Nussknacker side, we will be handling it in the following way:
1. check version of the data
2. decide what parser (or adapter, whatever) to user
3. convert the source format to the current, target format

Why do we have to do it in the two steps like here?

Reply:
No, the version check has to be made on the source side. What if source version is greater than the target one? In my approach source then lowers its version as much as possible.

Reply:
Ah, the requirement about migration from a higher version to a lower one. It's a pity we don't have the requirements listed :) 

Why the new endpoint, not the existing one?

Reply:
I dont get the question :(

Reply:
sorry, I was referring to the new version endpoint. We already have an endpoint which returns a Nu version (and some other things)

Reply:
Yes, but this endpoint has some legacy quirks and gave me weird responses. Are u Ok I'll go with my Version only and comment it should be removed when that more general one stabilize?

Reply:
what quirks exactly?